llm_bench/python/utils/model_utils.py

```diff
@@ -49,6 +50,16 @@ def set_default_param_for_ov_config(ov_config):
         ov_config['NUM_STREAMS'] = '1'
 
 
+def add_stateful_model_arguments(parser: argparse.ArgumentParser):
+    parser.add_argument(
+        '--stateful',
+        action='store_true',
+        default=None,
```

Comment:
No, it will return `None`, if you don't set it. In our case it means "use default". In convert.py it will activate stateless. In benchmark.py will take model as-is, which allows running both stateful and stateless models in benchmark.py without changing and requiring a particular mode.